File: Starware.DatingApp.UI/src/app/models/users/Message.ts
export interface Message {
    id: number;
    creationDate: Date;
    senderId: number;
    senderUsername: string;
    senderPhotoUrl: string;
    recipientId: number;
    recipientUsername: string;
    recipientPhotoUrl: string;
    content: string;
    dateReaded?: Date;
    recipientDeleted: boolean;
    senderDeleted: boolean;
}

File: Starware.DatingApp.Core/Domains/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Starware.DatingApp.Core.Domains
{
    public class Message : Entity
    {
        public int SenderId { get; set; }
        public string SenderUsername { get; set; }
        public AppUser Sender { get; set; }
        public int RecipientId { get; set; }
        public string RecipientUsername { get; set; }
        public AppUser Recipient { get; set; }
        public string Content { get; set; }
        public DateTime? DateReaded { get; set; }
        public bool RecipientDeleted { get; set; }
        public bool SenderDeleted { get; set; }

    }
}
